### PR TITLE
feat(ftux): create default endpoint after app deploys successfully

### PR DESCRIPTION
**Pull request by neurosnap**
https://user-images.githubusercontent.com/1940365/223149104-91b79431-2b91-47c0-9029-2400f3dfb240.mp4

### CODE REVIEW

File: src/projects/index.ts [hunk 1]
```diff
@@ -148,77 +177,177 @@ export const deployProject = thunks.create<CreateProjectSettingsProps>(
...
-        createAppOperation({
-          type: "configure",
-          appId,
-          env: envs.reduce(
-            (acc, env) => ({ ...acc, [env.key]: env.value }),
-            {},
-          ),
-        }),
-      ),
+    const env: { [key: string]: any } = {};
```
Comment (madhuravius) [inline src/projects/index.ts:196]: our backend supports anything besides string/integer for the env value (?). might wanna check this (we might only support string values, or string/int only)

File: src/deploy/endpoint/index.ts [hunk 2]
```diff
@@ -114,3 +136,106 @@ export const endpointEntities = {
...
+    const body = JSON.stringify({
+      app_id: appId,
+      service_id: serviceId,
+      platform: "alb",
+      type: "http_proxy_protocol",
+      status: "pending",
+      default: true,
+      acme: false,
+      internal: false,
+    });
```
Comment (joshraker) [inline src/deploy/endpoint/index.ts:158]: There's a few extra parameters here: `app_id`, `service_id` (pulled from the path), and `status`. They'll just be ignored so it's not a big deal unless we do start using them for some reason in the future. You can find the list of parameters expected on a create [here](https://github.com/aptible/deploy-api/blob/master/app/controllers/vhosts_controller.rb#L45-L49).

File: src/ui/pages/create-project-git.tsx [hunk 3]
```diff
@@ -426,6 +432,7 @@ const validateEnvs = (items: TextVal[]): ValidatorError[] => {
     }
   };
 
+  console.log(items);
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:435]: might wanna drop

File: src/ui/pages/create-project-git.tsx [hunk 4]
```diff
@@ -443,10 +450,190 @@ const useLatestCodeResults = (appId: string) => {
...
+};
+
+const idCreator = () => {
+  let id = 0;
+  return () => {
+    id += 1;
+    return id;
+  };
+};
+const createId = idCreator();
```
[the remark this answers is not in the document]
Comment (madhuravius) [reply src/ui/pages/create-project-git.tsx:501]: this is smarter than me, ignore

**Review by madhuravius — APPROVED** [review]

**Pull request merged**